sphinx/texinputs/sphinx.sty

```diff
@@ -397,6 +402,16 @@
 %          macro prefix   option prefix   legacy option    init value
 \spx@tempa{spx@pre@}      {pre_}          {verbatimborder} {0.4pt}
 \spx@tempa{spx@topic@}    {div.topic_}    {shadowrule}     {0.5pt}% mod. at 7.4.0
+\spx@tempa{spx@contents@} {div.contents_} {shadowrule}     {0.5pt}% 8.1.0
+\spx@tempa{spx@sidebar@}  {div.sidebar_}  {shadowrule}     {2pt}%   8.1.0
+\@nameuse{KV@sphinx@div.sidebar_border-bottom-width}{1pt}
```

Comment:
Not in favor of that. I count 7 uses and a macro would simply obfuscate and have no other usage whatsoever.

Reply:
Hmm actually I am a bit tired myself.  Forgot that this construct is used for example in 
```
\def\KV@sphinx@shadowrule #1{%
    \@nameuse{KV@sphinx@div.topic_border-width}{#1}%
    \@nameuse{KV@sphinx@div.contents_border-width}{#1}%
    \@nameuse{KV@sphinx@div.sidebar_border-width}{#1}%
}%
```
so it is not only "7 uses" because some are dynamical depending on whether legacy option ``shadowrule``  was made use of.  But this is purely internal business of Sphinx and in the end it will have to become ``\csname KV@sphinx@div.sidebar_border-width\endcsname`` etc... so using  a wrapper macro takes a bit of room in TeX hash table for no real benefit.